pyrtlnet/numpy_inference.py

```diff
@@ -201,21 +200,21 @@ def _run_layer(
         return layer_output.astype(np.int8)
 
     def preprocess_image(self, test_image: np.ndarray) -> np.ndarray:
-        """Preprocess the raw image data, as required by the quantized neural network.
+        """Preprocess the raw image data in the batch.
+        This is required by the quantized neural network.
 
-        This flattens the 2D image data into 1D, adds a batch dimension, and adjusts the
-        image data by ``input_scale`` and ``input_zero``.
+        This adjusts the batch image data by ``input_scale`` and ``input_zero``.
+        Then, it flattens the 2D image(s) into 1D and adds the batch dimension.
 
-        :param test_image: Image data to preprocess. This data should have already been
+        :param test_batch: Batch data to preprocess. This data should have already been
             normalized to [0.0, 1.0] and resized to 12×12, usually by
```

Comment:
This should also mention the shape of `test_batch` somewhere `(batch_size, 12, 12)`; it may not be obvious to the reader that the batch dimension is first